src/pycirclizely/parser/table.py

```diff
@@ -15,24 +15,35 @@ def __init__(
         table_data: str | Path | pd.DataFrame,
         *,
         delimiter: str = "\t",
+        color_cycler: ColorCycler | None = None,
     ):
-        """Parameters
-        ----------
-        table_data : str | Path | pd.DataFrame
-            Table file or Table DataFrame
-        delimiter : str, optional
-            Table file delimiter. By default, `tab` delimiter.
-
+        """
+        Args:
+            table_data: Table file or Table DataFrame.
+            delimiter: Table file delimiter. By default, `tab` delimiter.
+            color_cycler: Optional ColorCycler instance.
+                If None, creates a default one.
         """
         if isinstance(table_data, (str, Path)):
             table_data = pd.read_csv(table_data, sep=delimiter, index_col=0)
         self._dataframe = table_data
+        self._color_cycler = color_cycler or ColorCycler("T10")  # Default palette
 
     @property
     def dataframe(self) -> pd.DataFrame:
         """Table dataframe"""
         return self._dataframe
 
+    @property
+    def color_cycler(self) -> ColorCycler:
+        """Get the color cycler instance"""
+        return self._color_cycler
+
+    @color_cycler.setter
+    def color_cycler(self, cycler: ColorCycler):
+        """Set the color cycler instance"""
+        self._color_cycler = cycler
```

Comment:
this looks like has no effect. Consider removing it and renaming _color_cycler to color_cycler